packages/tooling/fast-tooling-react/src/form/form.tsx

```diff
@@ -99,6 +102,10 @@ class Form extends React.Component<
             props.messageSystem.add(this.messageSystemConfig);
         }
 
+        if (props.strings !== undefined && props.strings !== null) {
```

Comment:
in javascript you can force it to be a bool and then just do a falsey check:
```suggestion
        if (!!!props.strings) {
```

Reply:
Done